doc/api/events.markdown

```diff
@@ -137,10 +137,15 @@ Execute each of the listeners in order with the supplied arguments.
 Returns `true` if event had listeners, `false` otherwise.
 
 
+### emitter.listenerCount(type)
+
+Returns the number of listeners listening to the `type` of event.
```

Comment:
Since we're making a change, add a bullet that `type` is a `{String}`. As done in `'removeListener'`.